examples/basic-recipe.rs

```diff
@@ -29,21 +44,24 @@ fn main() {
     let mut awvc = blender.get_getter(outputs["ArticleWithVoteCount"]).unwrap();
 
     println!("Creating article...");
-    article
-        .put(vec![
-            1.into(),
-            "test title".into(),
-            "http://csail.mit.edu".into(),
-        ])
-        .unwrap();
-
-    println!("Casting votes...");
-    vote.put(vec![1.into(), 42.into()]).unwrap();
-    vote.put(vec![1.into(), 43.into()]).unwrap();
-    vote.put(vec![1.into(), 44.into()]).unwrap();
+    let aid = 1;
+    // Make sure the article exists:
+    if awvc.lookup(&aid.into(), true).unwrap().is_empty() {
+        println!("Creating new article...");
+        let title = "test title";
+        let url = "http://pdos.csail.mit.edu";
+        article
+            .put(vec![aid.into(), title.into(), url.into()])
+            .unwrap();
+    }
+
+    // Then create a new vote:
+    println!("Casting vote...");
+    let uid = time::get_time().sec;
```

Comment:
Why is the `time` crate needed for this? Why is `SystemTime::now().duration_since(UNIX_EPOCH).as_secs()` not sufficient using `std::time`?

Reply:
Good idea - fixed in https://github.com/mit-pdos/distributary/pull/37/commits/a6185971bdf70c230c4f16d56307cc509a0f30b1.